frontend/src/components/WorksheetListItem.js

```diff
@@ -75,7 +75,7 @@ function WorksheetListItem({
           <span className={styles.course_title}>{course.title}</span>
         </Col>
         {/* Hide Button */}
-        <Col xs="auto" className={'pl-0 pr-2 my-auto hidden'}>
```

Comment:
😂

Reply:
we don't talk about that

Reply:
lol i was laughing at the commit name, typescript/eslint/prettier be making errors out of everything